client/blocks/login/social.jsx

```diff
@@ -95,10 +88,14 @@ class SocialLoginForm extends Component {
 						onClick={ this.trackGoogleLogin } />
 				</div>
 
-				{ this.state.bearerToken && (
+				{ this.props.isSocialAccountCreating &&
+					<InfoNotice text={ this.props.translate( 'Creating your account…' ) } />
```

Comment:
I'm still not convinced about the idea of having a special block that is used to display a global notice. We can achieve exactly the same behavior using `componentWillReceiveProps` and a few lines of code.